apps/codeforafrica/src/lib/api.netlify-cms/getOffices.js

```diff
@@ -0,0 +1,25 @@
+import { join } from "path";
+
+import getCollectionData from "./getCollectionData";
+
+const officesDir = join(process.cwd(), "content/offices");
+
+export default function getOffices(fields) {
+  const offices = getCollectionData(officesDir, fields);
+  return offices.map((office) => {
+    return {
+      map: {
+        center: {
+          lat: JSON.parse(office.map.latitude),
+          lng: JSON.parse(office.map.longitude),
+        },
+        position: {
+          lat: JSON.parse(office.map.latitude),
+          lng: JSON.parse(office.map.longitude),
+        },
+      },
+      address: office.content,
```

Comment:
Not a biggie but since we're using `content` elsewhere for HTML content, we can just leave this as content as well.